### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ericgaspar**

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

File: scripts/install [hunk 1]
```diff
@@ -23,118 +22,151 @@ ynh_abort_if_errors
...
 # Use logrotate to manage application logfile(s)
 ynh_use_logrotate
 
 #=================================================
-# ADVERTISE SERVICE IN ADMIN PANEL
+# INTEGRATE SERVICE IN YUNOHOST
+#=================================================
+ynh_script_progression --message="Integrating service in YunoHost..." --weight=1
+
+yunohost service add $app --description="A short description of the app" --log="/var/log/$app/$app.log"
```
Comment (alexAubin) [inline scripts/install]:
```suggestion
yunohost service add $app --log="/var/log/$app/$app.log"
```

(We should either put an actual description, or no description at all, then the description from the systemd file shall be used automatically by yunohost)

File: scripts/restore [hunk 2]
```diff
@@ -3,126 +3,124 @@
...
 
 #=================================================
-# RESTORE SYSTEMD
+# INTEGRATE SERVICE IN YUNOHOST
 #=================================================
+ynh_script_progression --message="Integrating service in YunoHost..." --time --weight=1
 
-ynh_restore_file "/etc/systemd/system/$app.service"
-systemctl enable $app.service
+yunohost service add $app --description="A short description of the app" --log="/var/log/$app/$app.log"
```
Comment (alexAubin) [inline scripts/restore]:
```suggestion
yunohost service add $app --log="/var/log/$app/$app.log"
```

File: scripts/upgrade [hunk 3]
```diff
@@ -18,116 +18,143 @@ app=$YNH_APP_INSTANCE_NAME
...
+#if [ -z "$db_name" ]; then
+#	db_name=$(ynh_sanitize_dbid --db_name=$app)
+#	ynh_app_setting_set --app=$app --key=db_name --value=$db_name
+#fi
+
+# If final_path doesn't exist, create it
+#if [ -z "$final_path" ]; then
+#	final_path=/var/www/$app
+#	ynh_app_setting_set --app=$app --key=final_path --value=$final_path
+#fi
```
Comment (alexAubin) [inline scripts/upgrade]:
```suggestion
```

File: scripts/upgrade [hunk 4]
```diff
@@ -18,116 +18,143 @@ app=$YNH_APP_INSTANCE_NAME
...
 
 #=================================================
-# SETUP SYSTEMD
+# INTEGRATE SERVICE IN YUNOHOST
 #=================================================
+ynh_script_progression --message="Integrating service in YunoHost..." --time --weight=1
 
-# Create a dedicated systemd config
-ynh_systemd_config
+yunohost service add $app --description="A short description of the app" --log="/var/log/$app/$app.log"
```
Comment (alexAubin) [inline scripts/upgrade]:
```suggestion
yunohost service add $app --log="/var/log/$app/$app.log"
```

**Review by alexAubin — APPROVED** [review]

**Comment by alexAubin** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Comment by alexAubin** [comment]
!testme

File: README.md [hunk 5]
```diff
@@ -1,18 +1,48 @@
...
 
+## Overview
 
+Facette is a web application to display time series data from various sources — such as collectd, Graphite, InfluxDB or KairosDB — on graphs, designed to be easy to setup and to use.
 
+**Shipped version:** 0.5.1~ynh1
 
-**Upgrade this package:**  
-`sudo yunohost app upgrade --verbose example -u https://github.com/YunoHost-Apps/facette_ynh`
+**Demo:** https://demo.example.com
```
Comment (kay0u) [inline README.md]:
```suggestion
**Demo:** https://play.facette.io/browse/
```

File: README_fr.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+> *Ce package vous permet d'installer Facette rapidement et simplement sur un serveur YunoHost.
+Si vous n'avez pas YunoHost, regardez [ici](https://yunohost.org/#/install) pour savoir comment l'installer et en profiter.*
+
+## Vue d'ensemble
+
+Facette is a web application to display time series data from various sources — such as collectd, Graphite, InfluxDB or KairosDB — on graphs, designed to be easy to setup and to use.
+
+**Version incluse :** 0.5.1~ynh1
+
+**Démo :** https://demo.example.com
```
Comment (kay0u) [inline README_fr.md]:
```suggestion
**Démo :** https://play.facette.io/browse/
```

**Pull request merged**